### PR TITLE
feat(blocks): print pdf

### PR DESCRIPTION
**Pull request by EYHN**

### CODE REVIEW

**Comment by EYHN** [comment]
* **#7912** <a href="https://app.graphite.dev/github/pr/toeverything/blocksuite/7912?utm_source=stack-comment-icon" target="_blank">Graphite</a> 👈
* `master`

This stack of pull requests is managed by Graphite. <a href="https://stacking.dev/?utm_source=stack-comment">Learn more about stacking.</a>
<h2></h2>

 Join @EYHN and the rest of your teammates on <a href="https://graphite.dev?utm-source=stack-comment">Graphite <b>Graphite</b></a>

**Review by fourdim — COMMENTED** [review]
Why are we having too many important! in CSS?
If you are putting media print after the original CSS, I think it's possible to override it.

File: packages/blocks/src/_common/print-manager/print-manager.ts [hunk 1]
```diff
@@ -0,0 +1,124 @@
+export class PrintManager {
```
Comment (Saul-Mirone) [inline packages/blocks/src/_common/print-manager/print-manager.ts:1]: I think we should put this in `@blocksuite/affine-shared/utils`.

**Review by Saul-Mirone — APPROVED** [review]
LGTM \w little comments

**Review by Saul-Mirone — APPROVED** [review]
LGTM

File: packages/blocks/src/_common/print-manager/print-manager.ts (hunk 1 above)
Comment (EYHN) [reply packages/blocks/src/_common/print-manager/print-manager.ts]: done

**Comment by EYHN** [comment]
> Why are we having too many important! in CSS?
> If you are putting media print after the original CSS, I think it's possible to override it.

`!important` has been removed as much as possible

**Pull request merged**